mappings/net/minecraft/structure/WoodlandMansionGenerator.mapping

```diff
@@ -47,8 +47,40 @@ CLASS net/minecraft/class_3471 net/minecraft/structure/WoodlandMansionGenerator
 		FIELD field_15449 position Lnet/minecraft/class_2338;
 		FIELD field_15450 rotation Lnet/minecraft/class_2470;
 	CLASS class_3477 SecondFloorRoomPool
+	CLASS class_3478 FlagMatrix
+		FIELD field_15451 array [[I
+		FIELD field_15452 fallback I
+		FIELD field_15453 m I
+		FIELD field_15454 n I
```

Comment:
What are these?

Reply:
m and n are rows (n) and columns (m) as seen in computer programming problems. This is like indicating the room type I recall.

Reply:
Ah, this is a matrix.